index.bs

```diff
@@ -183,9 +183,13 @@ Integration with HTML {#html}
 
 This specification integrates with the [[!HTML]] specification by patching the algorithms below:
 
-Navigation response {#navigation-response}
+Speculative HTML parsing {#speculative-html-parsing}
 ----------
+At [=speculative fetch=], after the sub-step for the meta element "viewport" step add a new sub-step with the following:
+"A meta element whose name attribute is an ASCII case-insensitive match for "accept-ch" (only if this element is in the root Document and has not been modified by any script execution)."
```

Comment:
What does "modified by script execution" mean? How would you expect someone to implement that? If the intention is to go find all the places in an implementation where a document can be modified by script execution, and have them flip some sort of boolean that you later consult... you need to find all those places in the spec, and do so!